eth/v1/beacon_state.pb.gw.go

```diff
@@ -0,0 +1,3 @@
+// +build ignore
+
+package ignore
```

Comment:
Just wondering, what is up with this file?

Reply:
They're generated by gateway for some reason. Not sure.